configs/nextflow.hopper.config

```diff
@@ -0,0 +1,170 @@
+// Hopper specific configuration for micro wgs pipeline //
+
+singularity{
+  enabled = true
+  runOptions = '--bind /fs1/ --bind /local/'
+}
+
+process {
+	executor='slurm'
+	queue='grace-high'
+	time=2.h
```

Comment:
A bad default since this will punish processes in this pipeline that only takes a minute or so to run.

Reply:
Yeah exactly...